locales/en/common.json

```diff
@@ -1,12 +1,11 @@
 {
-    "home": "Home",
-    "about": "About Us",
-    "faq": "FAQ",
-    "media": "Media Inquires",
-    "add_story": "Add Your Story",
-    "analytics": "Analytics",
-    "email": "Email Us",
-    "translate": "Help Us Translate",
-    "made_with": "Made with love by Ontarians ❤️",
-    "tagline": "If our leaders won’t listen to the numbers, they must face our stories."
-}
\ No newline at end of file
+  "home": "Home",
+  "about": "About Us",
+  "faq": "FAQ",
+  "media": "Media",
```

Comment:
Media Inquiries is too long, it makes the nav bar wrap at certain screen widths. So, I reverted back to Media.